src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/LanguageServerFeatureOptions.cs

```diff
@@ -47,4 +47,10 @@ internal abstract class LanguageServerFeatureOptions
     public abstract bool UseRazorCohostServer { get; }
 
     public abstract bool DisableRazorLanguageServer { get; }
+
+    /// <summary>
+    /// Whether the "fuse" feature of razor is enabled, which uses the runtime output of razor in the IDE
+    /// instead of the design time output.
+    /// </summary>
+    public abstract bool EnableFuse { get; }
```

Comment:
I wanted to keep the terminology the same here, since we've been referring to it as such. I'm also open to just calling it `SuppressDesignTime` if people prefer. I just think from a tooling perspective it's a "feature" that results in `SuppressDesignTime` being set to true. I am also probably overthinking this